client/src/components/Channel.js

```diff
@@ -232,7 +232,7 @@ class Channel extends React.Component {
         transitionLeaveTimeout={0}
         >
         <div className="Controls" key="controls">
-          <SendMessage onSendMessage={this.sendMessage.bind(this)} key="SendMessage" theme={theme}/>
+          <SendMessage onSendMessage={this.sendMessage.bind(this)} key="SendMessage" theme={theme} useEmojis={this.state.appSettings.useEmojis}/>
           <Dropzone className="dropzone2" onDrop={this.onDrop.bind(this)} key="dropzone2">
```

Comment:
Excellent! Forgot to put this in the spec but this is exactly what should happen: if useEmojis option is on, display the preview, don't if not on.